android/BOINC/app/src/main/java/edu/berkeley/boinc/utils/BOINCUtils.java

```diff
@@ -61,7 +61,7 @@ public static String translateNetworkSuspendReason(Context ctx, int reason) {
 	
 	@SuppressLint("DefaultLocale")
 	public static String formatSize(double fBytesSent, double fFileSize) {
-		String buf = "";
+	    String buf;
```

Comment:
Could we just get rid of this variable at all? Then next if-else statements could be simplified.

Reply:
Done.